linodecli/output.py

```diff
@@ -134,15 +136,18 @@ def _table_output(
             ),
         )
 
-        tab = SingleTable(content)
+        tab = Table(*content[0])
```

Comment:
To preserve existing styling, these arguments are needed:

```suggestion
        tab = Table(*content[0], header_style="", box=box.SQUARE)
```